server/model/maintenance.js

```diff
@@ -0,0 +1,38 @@
+const dayjs = require("dayjs");
+const utc = require("dayjs/plugin/utc");
+let timezone = require("dayjs/plugin/timezone");
+dayjs.extend(utc);
+dayjs.extend(timezone);
+const { BeanModel } = require("redbean-node/dist/bean-model");
+
+class Maintenance extends BeanModel {
+
+    /**
+     * Return a object that ready to parse to JSON for public
+     * Only show necessary data to public
+     */
+    async toPublicJSON() {
+        return {
+            id: this.id,
+            title: this.title,
+            description: this.description,
+            start_date: this.start_date,
+            end_date: this.end_date
```

Comment:
For better git diff in next updates to this object
```suggestion
            end_date: this.end_date,
```